lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.html

```diff
@@ -57,6 +57,10 @@
                 [root]="breadcrumbFolderTitle"
                 data-automation-id="content-node-selector-content-breadcrumb">
             </adf-dropdown-breadcrumb>
+            <div
+                data-automation-id="content-node-selector-selected-counter">
```

Comment:
is the counter visible on the local upload tab too? 
I was expecting to have a directive